annotations/views.py

```diff
@@ -158,6 +158,14 @@ def user_settings(request):
     })
     return HttpResponse(template.render(context))
 
+@login_required
```

Comment:
The view should be publicly available. Right now, only logged-in users can see it.